packages/react/src/components/SheetOverlay/ContentEditable.tsx

```diff
@@ -13,70 +13,78 @@ type ContentEditableProps = Omit<
   allowEdit?: boolean;
 };
 
-class ContentEditable extends React.Component<ContentEditableProps> {
-  lastHtml?: string = undefined;
+const ContentEditable = ({ ...props }: ContentEditableProps) => {
+  const [lastHtml, setLastHTML] = useState("");
+  let root: HTMLDivElement | null | unknown = useRef<HTMLDivElement | null>(
+    null
+  );
+  const { autoFocus, initialContent, onChange } = props;
 
-  root: HTMLDivElement | null = null;
-
-  componentDidMount() {
-    const { autoFocus, initialContent } = this.props;
-    if (autoFocus) {
-      this.root?.focus();
+  useEffect(() => {
```

Comment:
<img width="596" alt="image" src="https://user-images.githubusercontent.com/84694078/210714573-f246fb2a-f064-4789-a346-d1c97fc3e512.png">

Reply:
thank you. 

BTW. have eslint warning
![image](https://user-images.githubusercontent.com/1460479/211155303-25a501e5-d945-4e51-9132-ab6b7def9faa.png)

Reply:
You can just change the effect to this 
```
  useEffect(() => {
    if (autoFocus) {
      root.current?.focus();
    }
  }, [autoFocus]);

  useEffect(() => {
    if (initialContent && root.current != null) {
      root.current.innerHTML = initialContent;
    }
  }, [initialContent]);
```
I wrote this according to the recommendations of the official documents
[Tip: Use Multiple Effects to Separate Concerns](https://reactjs.org/docs/hooks-effect.html#tip-use-multiple-effects-to-separate-concerns)
Then there is no need to detect first render because we have separated them into different effects.

And about useRef, you can check [useRef](https://reactjs.org/docs/hooks-reference.html#useref), i think we can just follow the official documents, use it like this 
``` 
const root = useRef<HTMLDivElement | null>(null); 
...
...
     ref={(e) => {
        root.current = e;
        innerRef?.(e);
      }}
...
...
```
or maybe you use let to declare it for other reasons(if so, you can let me know how did you think about it)

Reply:
yeah. ya right. thank you. it sounds correct. 
fixed as u said.